modules/nodes-scheduler-pod-affinity-configuring.adoc

```diff
@@ -26,31 +32,48 @@ spec:
   - name: security-s1
     image: docker.io/ocpqe/hello-pod
 ----
-
-. When creating other pods, edit the `Pod` spec as follows:
 +
-.. Use the `podAffinity` stanza to configure the `requiredDuringSchedulingIgnoredDuringExecution` parameter or `preferredDuringSchedulingIgnoredDuringExecution` parameter:
+.. Create the pod.
 +
-.. Specify the key and value that must be met. If you want the new pod to be scheduled with the other pod, use the same `key` and `value` parameters as the label on the first pod.
+[source,terminal]
+----
+$ oc create -f <pod-spec>.yaml
+----
+
+. When creating other pods, configure the following parameters to add the affinity:
++
+.. Create a YAML file with the following:
```

Comment:
```suggestion
.. Create a YAML file with the following content:
```
Per IBM Style, always include a noun after "following" unless you're using "following" as a noun (e.g., "that band has a huge following"). See the entry for "following" on the Word Usage page: https://www.ibm.com/docs/en/ibm-style?topic=word-usage